packages/embark-typings/src/embark.d.ts

```diff
@@ -6,7 +6,10 @@ export interface Events {
   request: any;
   emit: any;
   once: any;
-  setCommandHandler(name: string, callback: (options: any, cb: () => void) => void): void;
+  setCommandHandler(
+    name: string,
+    callback: (options: any, cb: () => void) => void,
+  ): void;
```

Comment:
I assume this change is unrelated to this PR's scope and prettier did its thing?